cgale/index.html

```diff
@@ -83,6 +83,7 @@
                     <div class="col-lg-12">
                         <p>
                             Copyright © Connie Gale 2016
+                            Created by AJ Donald
```

Comment:
This is the only change in the only file that is causing problems on this commit. What should be done about it?